index.html

```diff
@@ -10,8 +10,68 @@
     <link rel="stylesheet" href="style.css">
 </head>
 <body>
-    <div class="container-fluid">
-        <button class="btn btn-primary btn_style" onclick="window.location.href='menu.html'" id="cardapio_home">Cardápio</button>
+    <header id="container-menu">
+        <ul>
+            <li><a href="" class="logo"></a></li>
+            <li class="header-menu"><a href="">Quem somos</a></li>
+            <li class="header-menu"><a href="">Unidades</a></li>
+            <li class="header-menu"><a href="">Cardápios</a></li>
+            <a href="" class="header-menu-cart"></a>
+        </ul>
+    </header>
+    <main>
+        <img src="assets/img/Villa-eleganza.png" alt="" class="img-villa-eleganz-main">
```

Comment:
It's a little off figma. Plus, in bigger screens it breaks a little (see those little grey columns?). Maybe we could do it inside a div, applying width:1356px (like figma) to img and background-color: #AB4914 to div.

<img src="https://github.com/armentanoc/villa_eleganza_menu/assets/88147887/3a714f19-4fee-4336-bd4d-b600527ba938" width="500px">